flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+    "only-arches": [
+      "x86_64"
+    ],
+    "automerge-flathubbot-prs": true
```

Comment:
This is a bad idea, you already ran into one error using precompiled binaries. Builds will get pushed to users without any testing

Reply:
It's going to be a part of an automated distribution. 

As for the errors, existing one gonna to be solved. Ongoing will fail the build and prevent the package from its distribution.

Reply:
Got it, replaced by `false`